app/views/idv/inherited_proofing/verify_wait.html.erb

```diff
@@ -14,3 +14,4 @@
 
 <p><%= t('inherited_proofing.info.retrieval_time') %></p>
 <p><%= t('inherited_proofing.info.retrieval_thanks') %></p>
+<%= render 'idv/inherited_proofing/cancel', step: 'verify_info' %>
```

Comment:
Would we expect the step here to be `verify_info`, or should it be `verify_wait` ?

Reply:
This is an async view; if we cancel, we want to go back to the previous step where the user would need to check confirmation checkbox and be allowed to start the process over.